Library/Homebrew/debrew.rb

```diff
@@ -99,7 +99,7 @@ def self.debrew
   end
 
   def self.debug(exception)
-    raise(exception) if !active? || !debugged_exceptions.add?(exception) || !try_lock
+    raise(exception) if !active? || !debugged_exceptions.add?(exception) || !mu_try_lock
```

Comment:
This switches to using the `mu_` methods bc sorbet doesn't recognize the aliased versions, but I believe they are equivalent:

```
[1] brew(main)> require 'debrew'
=> true
[2] brew(main)> $ Debrew.try_lock

From: /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/mutex_m.rb:86:
Owner: #<Class:Debrew>
Visibility: public
Signature: try_lock()
Number of lines: 3

def mu_try_lock
  @_mutex.try_lock
end
```